setup.py

```diff
@@ -21,8 +21,9 @@
         'requests==2.21.0',
         'paramiko==2.4.2',
         'pyyaml>=4.2b1',
-        'jinja2==2.10.1',
-        'junitparser==1.4.0'
+        'jinja2',
```

Comment:
why did you remove the version number here ?

Reply:
There was a conflict with jinja2 Version, where was in a dependent package required for jinja_markdown was not getting installed with current version of jinja2 present. So removing the version solved the dependency issue.
``` ERROR: Cannot install -r requirements.txt (line 1) and cephci because these package versions have conflicting dependencies.

The conflict is caused by:
    cephci 0.1 depends on jinja2==2.10.1
    jinja-markdown  depends on Jinja2~=2.11
```